store-tests

```diff
@@ -44,6 +44,7 @@ from task import github
 # not ok 78 testNodeNavigation (check_openshift.TestOpenshift) # duration: 172s
```

Comment:
That comment is not actually true any more -- can you please update it to the current format?